lib/ecto/changeset.ex

```diff
@@ -4370,6 +4359,17 @@ defimpl Inspect, for: Ecto.Changeset do
     end)
   end
 
+  defp get_redacted_fields(%type{}) do
+    type
+    |> function_exported?(:__schema__, 1)
+    |> get_redacted_fields(type)
+  end
+
+  defp get_redacted_fields(_), do: []
+
+  defp get_redacted_fields(true, type), do: type.__schema__(:redact_fields)
+  defp get_redacted_fields(_, _type), do: []
```

Comment:
Now that implementation is moved to the schema module, I figured I'd leave this here as cleanup. 😄

Reply:
Let's keep it as it was, as it is an unrelated change as of now (and I am not particularly sure it is cleaner, as it has more indirection). :)

Reply:
Done!